tests/unit/test_fileio.py

```diff
@@ -261,6 +279,7 @@ def test_write(self):
             None,
             NUM_RETRIES,
             chunk_size=chunk_size,
+            retry=None,
             **upload_kwargs
         )
         upload.transmit_next_chunk.assert_called_with(transport)
```

Comment:
Do we need a testcase for `write` that assures that a non-default `retry` is passed through?

Reply:
OK, added test as `test_forced_default_retry`